core/templates/dev/head/components/skills-mastery-list/skills-mastery-list.directive.html

```diff
@@ -0,0 +1,107 @@
+<div class="oppia-skills-list-container">
+  <md-card class="oppia-page-card oppia-long-text">
+    <div class="skills">
```

Comment:
name this class such that it starts with `oppia-` to make it specific. ditto elsewhere

Reply:
Done.